numpyro/primitives.py

```diff
@@ -42,6 +42,9 @@ def apply_stack(msg):
 
 class Messenger(object):
     def __init__(self, fn=None):
+        if fn is not None and not callable(fn):
+            raise ValueError("Expected `fn` to be callable; instead found type(fn) = {}."
```

Comment:
nit: can we change `to be callable` --> `to be a Python callable object` for clarity?

Reply:
thanks